src/core/tests/test_icon.py

```diff
@@ -12,9 +13,25 @@ def setUp(self):
 
         self.test_path = "Example.bmp"
         self.icon = toga.Icon(self.test_path)
+        self.resource_icon = toga.Icon(self.test_path, system=True)
 
     def test_icon_bind(self):
         self.assertEqual(self.icon._impl, None)
         self.icon.bind(factory=toga_dummy.factory)
         self.assertEqual(self.icon._impl.interface, self.icon)
         self.assertEqual(self.icon.path, self.test_path)
+
+    def test_icon_file(self):
+        """
+        Validate filename property
+
+        :return: None
```

Comment:
Please convert these to Google Style Docstrings

Reply:
done!